src/presentation/http/router/note.ts

```diff
@@ -316,9 +319,12 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
   }, async (request, reply) => {
     const noteId = request.note?.id as number;
     const content = request.body.content;
+    const noteTools = request.body.noteTools;
 
     const note = await noteService.updateNoteContentById(noteId, content);
 
+    await noteService.updateNoteToolsById(noteId, noteTools);
```

Comment:
seems like you can use the single update for content and tools instead of 2 separate